docs/zkapps/tutorials/09-recursion.mdx

```diff
@@ -16,12 +16,6 @@ keywords:
   - mina
 ---
 
-:::info
```

Comment:
Please revert this. See [internal discussion](https://o1-labs.slack.com/archives/C0206B5S80Y/p1714739315360389?thread_ts=1714734545.985919&cid=C0206B5S80Y).